app/database/MetricsDB.scala

```diff
@@ -56,5 +57,58 @@ class MetricsDB(implicit val db: Database) {
       }
 
   def getDistinctNewspaperBooks: Either[ProductionMetricsError, Seq[Option[String]]] =
-    await(db.run(metricsTable.filter(book => !book.newspaperBook.isEmpty && book.newspaperBook =!= "").map(_.newspaperBook).distinct.result))
+    await(db.run(metricsTable.filter(!_.newspaperBook.isEmpty).map(_.newspaperBook).distinct.result))
+
+  def getGroupedWordCounts(implicit filters: MetricsFilters): Either[ProductionMetricsError, List[GroupedWordCount]] = {
+
+    val lowerBoundsToUpperBounds: Map[Int, Int] = Map(
+      0 -> 349,
+      350 -> 649,
+      650 -> 899
+    )
+
+    awaitWithTransformation(db.run(metricsTable
+      .filter(MetricsFilters.metricFilters)
+      .map( metric =>
+        Case
```

Comment:
I would like to refactor this to generate the If statements from the `lowerBoundsToUpperBounds` map but I think this is something for a separate PR if I get the time!